skbio/stats/tests/test_composition.py

```diff
@@ -352,22 +352,23 @@ def test_ilr_inv(self):
     def test_ilr_basis_isomorphism(self):
         # tests to make sure that the isomorphism holds
         # with the introduction of the basis.
-        basis = np.array([[0.80442968, 0.19557032]])
+        basis = np.array([[0.70710677, -0.70710677]])
```

Comment:
I'm assuming it's correct that the code changes have resulted in value changes?

Reply:
Correct, since we are now using log probabilities instead of probabilities.  This is key for numerical stability.

I've reverted this test with the original data and added a `clr` transform to better showcase the change of coordinates.